tests/common/include/tests/common/test_toy_detector.hpp

```diff
@@ -293,33 +298,33 @@ inline bool test_toy_detector(
     // Test the links in the volumes
     test_volume_links(vol_itr, 1u, index, sf_finder_link);
 
-    // Check the trapezoid modules
-    range = {16u, 124u};
-    test_module_links(vol_itr->index(),
-                      surfaces.begin() + static_cast<std::ptrdiff_t>(range[0]),
-                      range, range[0], {mask_ids::e_trapezoid2, 0u},
-                      {material_ids::e_slab, 16u}, pixel_mat,
-                      {vol_itr->index()});
-
-    // Check link of surfaces in surface finder
-    // test_surfaces_grid(vol_itr, sf_finders, surfaces, range);
-
     // Check links of portals
     // cylinder portals
-    range = {124u, 126u};
+    range = {16u, 18u};
```

Comment:
The portals are now at the beginning of the range (when visiting the surfaces in the volume, the brute force method that holds the portals is queried first), so the indices switched